runtime/rasdump/dmpsup.c

```diff
@@ -1365,7 +1366,72 @@ initSystemInfo(J9JavaVM *vm)
 			}
 		}
 	}
-	appendSystemInfoFromFile(vm, J9RAS_SYSTEMINFO_CORE_PATTERN, J9RAS_CORE_PATTERN_FILE);
+	{
+		J9RASSystemInfo* corePatternInfo = appendSystemInfoFromFile(vm, J9RAS_SYSTEMINFO_CORE_PATTERN, J9RAS_CORE_PATTERN_FILE);
+		if ( corePatternInfo != NULL ) {
```

Comment:
Throughout:
* space before, not after `*` (in pointer declarations)
* constants on left side of `==` or `!=`
* no spaces inside `(` `)`
* parentheses missing in compound conditions